resources/leiningen/new/luminus/oauth/src/oauth.clj

```diff
@@ -0,0 +1,44 @@
+(ns <<project-ns>>.oauth
+  (:require [oauth.client :as oauth]
+            [config.core :refer [env]]
+            [clojure.tools.logging :as log]))
+
+;;Twitter used as an example, replace these URIs with the OAuth provider of your choice
+
+(def request-token-uri
+  "https://api.twitter.com/oauth/request_token")
```

Comment:
I think this should come from env by default. So, it would be better to add those as keys in `env/dev|prod/resources/config.edn` and then read them from there.